tests/playwright/src/specs/compose-onboarding-smoke.spec.ts

```diff
@@ -16,11 +16,8 @@
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
 
-import { afterEach } from 'node:test';
```

Comment:
interesting here, it means that the afterEach is probably never called today

Reply:
vitest analyze only the vitest afterEach

Reply:
oh I see, that is interesting.